sample/SampleEndpointApp/Endpoints/Authors/Get.cs

```diff
@@ -1,36 +1,39 @@
-﻿using Ardalis.ApiEndpoints;
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Ardalis.ApiEndpoints;
 using AutoMapper;
 using Microsoft.AspNetCore.Mvc;
 using SampleEndpointApp.DomainModel;
 
-namespace SampleEndpointApp.Endpoints.Authors;
```

Comment:
Why have you reverted to block-scoped namespaces here? Are they also block-scoped in the rest of the files (cannot remember)

Reply:
Probably chose poorly on the massive merge conflict I was dealing with. Sigh. I'll try to sort it out.

Reply:
At least it's only one file. What merge conflict BTW? I thought everything is already in `main`